int64 reset
---
string status
float64 deformation
float64 total_deformation
